lib/ssl/src/tls_connection_1_3.erl

```diff
@@ -237,61 +237,94 @@ user_hello({call, From}, cancel, State) ->
     ssl_gen_statem:handle_own_alert(?ALERT_REC(?FATAL, ?USER_CANCELED, user_canceled),
                                     ?FUNCTION_NAME, State);
 user_hello({call, From}, {handshake_continue, NewOptions, Timeout},
-           #state{static_env = #static_env{role = Role},
-                  handshake_env = #handshake_env{hello = Hello},
+           #state{static_env = #static_env{role = client = Role},
+                  handshake_env = HSEnv,
                   ssl_options = Options0} = State0) ->
-    Options = ssl:handle_options(NewOptions, Role, Options0#{handshake => full}),
+    Options= ssl:handle_options(NewOptions, Role, Options0),
```

Comment:
if you would be making a next version, consider replacing all `Options=` with `Options =`.
Ignore otherwise.

Reply:
I will fix this, I agree it is more readable with the space ( I usually do that, I consider it a typo).